### ISSUE TITLE
Debian/ALSA unable to init with PulseAudio disable

### ISSUE THREAD

**Issue by JulienH2000** [RustAudio/cpal#912]
Using Debian 12 / ALSA,
I'm unable to use my alsa device with pulseaudio is disable, even though I see the device in the alsamixer.
Code is panicking when unwrapping default_output_device() on the Alsa Host.
Returns error : ALSA lib pcm_dmix.c:1000: (snd_pcm_dmix_open) unable to open slave.
Seems to work fine with pulseaudio is running.
Thanks

**Comment by wgibbs-rs** [comment]
I was not able to reproduce this issue. I ran the CPAL examples on Debian 13.0.0, without PulseAudio enabled (or even installed in Docker)

**Comment by roderickvd** [comment]
Yes, please provide the full output of `aplay -L` and your `asoundrc` or similar files to have a chance of understanding why your `dmix` is not available. But I expect a system setup issue, not a cpal issue. 

Closing for now pending further feedback.
